app/konnect/runtime-manager/runtime-instances/gateway-runtime-aws.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Set up a Kong Gateway Runtime with AWS
+no_version: true
+content_type: how-to
+alpha: true
+---
+
+Set up a AWS runtime instance through the
+[{{site.konnect_short_name}} Runtime Manager](/konnect/runtime-manager) and
+configure your instance to accept configuration from
+{{site.konnect_short_name}}. The Runtime Manager keeps track of all runtime
+instances associated with the {{site.konnect_saas}} account.
+
+Runtime Manager provides a pre-populated template for a runtime instance in AWS. This template creates the following resources in AWS:
+* Amazon VPC along with internet gateway 
+* Secret
+* Amazon EC2 instances (key pair, role, profile)
+* Auto Scaling group
+* Network Load Balancer
+* Optional: CloudWatch log group 
+* Optional: Redis 
+
+{:.note}
+> **Note:** Kong does not host runtimes. You must install and host your own
+runtime instances.
```

Comment:
```suggestion
```
Removing this, since it's a page about setting up a runtime in another cloud provider. It doesn't make sense to say "you must install and host your own" when they're using a cloud provider.